test-helpers/src/main/java/cz/xtf/testhelpers/image/ImageMetadata.java

```diff
@@ -120,4 +124,42 @@ public String command() {
 
 		return result;
 	}
+
+	private static final class ImageStreamFailFastCheck implements FailFastCheck {
```

Comment:
This inner class is used twice (same code), could it be separated into some "failfast" package and used from there.

Reply:
It is very specific to the case. I would not even add it into `ImageMetadata` class, however someone might benefit from this. Even us, until `DockerImageMetadata` class is used. Moreover `ImageMetadata` is deprecated and after updating policy it can be removed.

Reply:
So If you really insist, I can move it to somewhere else.

Reply:
Will be handled as part of https://github.com/xtf-cz/xtf/issues/378